bencher/bench_vars.py

```diff
@@ -76,14 +73,16 @@ def param_hash(param_type: Parameterized, hash_value: bool = True, hash_meta: bo
 class ParametrizedSweep(Parameterized):
     """Parent class for all Sweep types that need a custom hash"""
 
-    def hash_custom(self) -> int:
+    def hash_custom(self) -> str:
+        """A hash function that avoids the PYTHONHASHSEED 'feature' which returns a different hash value each time the program is run"""
         return param_hash(self, True, False)
 
 
 class ParametrizedOutput(Parameterized):
     """Parent class for all Output types that need a custom hash"""
 
-    def hash_custom(self) -> int:
+    def hash_custom(self) -> str:
```

Comment:
ParametrizedOutput and ParametrizedSweep have the same class definition - are we able to remove duplication here?